plenum/test/pool_transactions/helper.py

```diff
@@ -363,19 +367,26 @@ def sdk_add_new_node(looper,
     return nym_request, named_did
 
 
-async def prepare_node_request(new_node_name, steward_did, clientIp,
-                               clientPort, nodeIp, nodePort, bls_key, sigseed):
-    data = {
-        'alias': new_node_name,
-        'client_ip': clientIp,
-        'client_port': clientPort,
-        'node_ip': nodeIp,
-        'node_port': nodePort,
-        'services': ["VALIDATOR"],
-        'blskey': bls_key
-    }
-    nodeSigner = SimpleSigner(seed=sigseed)
-    destination = nodeSigner.identifier
+async def prepare_node_request(steward_did, new_node_name=None, clientIp=None,
+                               clientPort=None, nodeIp=None, nodePort=None, bls_key=None,
+                               sigseed=None, destination=None):
+    if (sigseed is None and destination is None):
+        raise AttributeError('You must specify destination directly or using sigseed')
+    if (sigseed is not None and destination is not None):
+        raise AttributeError('You must use only one of: sigseed or destination')
```

Comment:
These two conditions are simply XOR operation and it would be easier to read and understand one operation.
Rewrite it to something like this
none_sigseed = segseed is None
none_dest = dest is None
if none_sigseed == none_dest : error you should provide only one of ...

Reply:
Done